app/controllers/ops_controller/settings/label_tag_mapping.rb

```diff
@@ -12,12 +12,16 @@ module OpsController::Settings::LabelTagMapping
   # In any case, this requires different providers use disjoint sets of strings.
   MappableEntity = Struct.new(:prefix, :model)
 
+  ALL_ENTITIES = "_all_entities_"
+
   MAPPABLE_ENTITIES = {
     # TODO: support per-provider "All Amazon" etc?
     # Currently we have only global "All".
     # Global "All" categories are named "kubernetes::..." for backward compatibility.
     nil                   => MappableEntity.new("kubernetes::",
                                                 nil),
+    ALL_ENTITIES          => MappableEntity.new(nil,
+                                                "All Entities"),
     "Vm"                  => MappableEntity.new("amazon:vm:",
                                                 "ManageIQ::Providers::Amazon::CloudManager::Vm"),
```

Comment:
Not related to this change at all, but this part doesn't look very pluggable, I'll add a task to our pluggability checklist